src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -611,13 +627,13 @@ public function next_tag( $query = null ): bool {
 	 * semantic rules for text nodes. For access to the raw tokens consider using
 	 * WP_HTML_Tag_Processor instead.
 	 *
-	 * @since 6.5.0 Added for internal support; do not use.
+	 * @since 6.7.0 Added for internal support; do not use.
 	 *
 	 * @access private
 	 *
 	 * @return bool
 	 */
-	public function next_token(): bool {
+	private function _next_token(): bool {
```

Comment:
next time or in a follow-up, would you be open to using `base_class_next_token()` here, or even something more descriptive like `next_visitable_token()` or `source_next_events()`?

so far this is the first underscore'd function name and it sticks out. I guess `base_class` isn't great here because of the Tag Processor, but something like "sourcing visitable tokens" or "prime event queue" could hint at what the function is doing.

Reply:
Implemented in https://github.com/WordPress/wordpress-develop/pull/7730